Description: fix some atlas tools stuff, make it use fonts from the system. DroidSansThai.ttf maybe is better than NotoSansThai-Regular.ttf, but it's not in the repository anymore.
Index: ppsspp-git/ext/native/tools/atlastool.cpp
===================================================================
--- ppsspp-git.orig/ext/native/tools/atlastool.cpp	2015-09-21 16:23:12.212560740 -0300
+++ ppsspp-git/ext/native/tools/atlastool.cpp	2015-09-21 16:23:12.212560740 -0300
@@ -11,7 +11,7 @@
 // line height
 // dist-per-pixel
 
-#include <libpng17/png.h>
+#include <libpng/png.h>
 #include <ft2build.h>
 #include FT_FREETYPE_H
 #ifdef _WIN32
Index: ppsspp-git/atlasscript.txt
===================================================================
--- ppsspp-git.orig/atlasscript.txt	2015-09-21 16:23:12.212560740 -0300
+++ ppsspp-git/atlasscript.txt	2015-09-21 16:23:12.212560740 -0300
@@ -1,6 +1,11 @@
 2048
-font UBUNTU24 assets/Roboto-Condensed.ttf UWER 34 -2
-font UBUNTU24 C:/Windows/Fonts/ARIALUNI.ttf UWEhkcRGHKVTe 30 0
+font UBUNTU24 /usr/share/fonts/truetype/roboto/RobotoCondensed-Regular.ttf UWER 34 -2
+font UBUNTU24 /usr/share/fonts/truetype/liberation/LiberationSans-Regular.ttf UWERG 30 0
+font UBUNTU24 /usr/share/fonts/truetype/dejavu/DejaVuSans.ttf e 30 0
+font UBUNTU24 /usr/share/fonts/truetype/droid/DroidSansJapanese.ttf hkc 30 0
+font UBUNTU24 /usr/share/fonts/truetype/droid/DroidSansHebrew-Regular.ttf H 30 0
+font UBUNTU24 assets/NotoSansThai-Regular.ttf T 30 0
+font UBUNTU24 /usr/share/fonts/truetype/unfonts-core/UnDotum.ttf K 30 0
 image I_SOLIDWHITE white.png copy
 image I_CROSS source_assets/image/cross.png copy
 image I_CIRCLE source_assets/image/circle.png copy
Index: ppsspp-git/atlasscript_lowmem.txt
===================================================================
--- ppsspp-git.orig/atlasscript_lowmem.txt	2015-09-21 16:23:12.212560740 -0300
+++ ppsspp-git/atlasscript_lowmem.txt	2015-09-21 16:23:12.212560740 -0300
@@ -1,5 +1,5 @@
 512
-font UBUNTU24 assets/Roboto-Condensed.ttf U 34 -2
+font UBUNTU24 /usr/share/fonts/truetype/roboto/RobotoCondensed-Regular.ttf U 34 -2
 image I_SOLIDWHITE white.png copy
 image I_CROSS source_assets/image/cross.png copy
 image I_CIRCLE source_assets/image/circle.png copy
Index: ppsspp-git/build_ppgeatlas.sh
===================================================================
--- ppsspp-git.orig/build_ppgeatlas.sh	2015-09-21 16:23:12.212560740 -0300
+++ ppsspp-git/build_ppgeatlas.sh	2015-09-21 16:23:12.212560740 -0300
@@ -1 +1 @@
-./atlastool ppge_atlasscript.txt ppge && mv ppge_atlas.cpp ppge_atlas.h Core/Util && cp ppge_atlas.zim assets && mv ppge_atlas.zim android/assets
+./ext/native/tools/build/atlastool ppge_atlasscript.txt ppge && mv ppge_atlas.cpp ppge_atlas.h Core/Util && mv ppge_atlas.zim assets
Index: ppsspp-git/ppge_atlasscript.txt
===================================================================
--- ppsspp-git.orig/ppge_atlasscript.txt	2015-09-21 16:23:12.212560740 -0300
+++ ppsspp-git/ppge_atlasscript.txt	2015-09-21 16:23:12.212560740 -0300
@@ -1,5 +1,9 @@
 2048
-font UBUNTU24 C:/Windows/Fonts/KozGoPro-Medium.otf UWkhcsGR 24
+font UBUNTU24 /usr/share/fonts/truetype/droid/DroidSans.ttf UWR 24
+font UBUNTU24 /usr/share/fonts/truetype/dejavu/DejaVuSans.ttf G 24
+font UBUNTU24 /usr/share/fonts/truetype/nanum/NanumGothic.ttf K 24
+font UBUNTU24 /usr/share/fonts/truetype/droid/DroidSansJapanese.ttf hkc 24
+font UBUNTU24 /usr/share/fonts/truetype/mona/mona.ttf s 24
 image I_CROSS source_assets/image/cross.png copy
 image I_CIRCLE source_assets/image/circle.png copy
 image I_SQUARE source_assets/image/square.png copy
Index: ppsspp-git/buildatlas.sh
===================================================================
--- ppsspp-git.orig/buildatlas.sh	2015-09-21 16:23:12.212560740 -0300
+++ ppsspp-git/buildatlas.sh	2015-09-21 16:23:12.212560740 -0300
@@ -1,4 +1,4 @@
-./ext/native/tools/build/atlastool atlasscript.txt ui 8888 && cp ui_atlas.zim assets && cp ui_atlas.zim android/assets && cp ui_atlas.zim.png ui_atlas_high.zim.png && mv ui_atlas.cpp ui_atlas_highmem.cpp && mv ui_atlas.h UI
+./ext/native/tools/build/atlastool atlasscript.txt ui 8888 && cp ui_atlas.zim assets && cp ui_atlas.zim.png ui_atlas_high.zim.png && mv ui_atlas.cpp ui_atlas_highmem.cpp && mv ui_atlas.h UI
 ./ext/native/tools/build/atlastool atlasscript_lowmem.txt ui 8888 && mv ui_atlas.zim assets/ui_atlas_lowmem.zim && mv ui_atlas.cpp ui_atlas_lowmem.cpp
 diff -I'^//.*' -D USING_QT_UI ui_atlas_highmem.cpp ui_atlas_lowmem.cpp > UI/ui_atlas.cpp
-rm ui_atlas_highmem.cpp ui_atlas_lowmem.cpp ui_atlas.h
+rm -f ui_atlas_highmem.cpp ui_atlas_lowmem.cpp ui_atlas.h
Index: ppsspp-git/ext/native/tools/CMakeLists.txt
===================================================================
--- ppsspp-git.orig/ext/native/tools/CMakeLists.txt	2015-09-21 16:20:29.777890333 -0300
+++ ppsspp-git/ext/native/tools/CMakeLists.txt	2015-09-21 16:24:04.985348717 -0300
@@ -45,11 +45,11 @@
 add_subdirectory(../math math)
 add_subdirectory(../util util)
 add_subdirectory(../ext/rg_etc1 rg_etc1)
-add_subdirectory(../ext/libpng17 png17)
+add_subdirectory(../ext/libpng png)
 
 
 add_executable(atlastool atlastool.cpp)
-target_link_libraries(atlastool freetype util image png17 z rg_etc1 file base)
+target_link_libraries(atlastool freetype util image png z rg_etc1 file base)
 
 add_executable(zimtool zimtool.cpp)
-target_link_libraries(zimtool freetype image png17 z rg_etc1 file base)
+target_link_libraries(zimtool freetype image png z rg_etc1 file base)
